src/fidesops/models/fidesops_user.py

```diff
@@ -48,3 +57,13 @@ def credentials_valid(self, password: str) -> bool:
         )
 
         return provided_password_hash == self.hashed_password
+
+    def update_password(self, db: Session, new_password: str) -> None:
```

Comment:
It would be good to have some validation in this flow such that the old password is verified before the password is updated to the new password. Another take on this could be that so long as the endpoint is protected with the right scopes, and that we're validating that the token's user and user who's password is being updated matches, then that's enough.

Reply:
There's also discussion over adding this validation in follow-up https://github.com/ethyca/fidesops/issues/404